### DISCUSSION TITLE
[FEATURE REQUEST] is there a way to identify source_region and destination region when creating AWS RDS read replica with  salt.modules.boto_rds.create_read_replica module?

### DISCUSSION THREAD

**Discussion by kvgarnet** [saltstack/salt#67333] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
based on https://ansible-cn.readthedocs.io/en/latest/ref/modules/all/salt.modules.boto_rds.html
When I tried to run salt cmd  to create a RDS replica as 
`salt-call boto_rds.create_read_replica   pdbgf01-qa1-replica  pdbgf01-qa1 
`
seems there is only one **region** parameter.
based on https://docs.aws.amazon.com/cli/latest/reference/rds/create-db-instance-read-replica.html
seems that region params is for the destination region of the read replica, there is a **--source-region** in AWS CLI to note the the source region of the DB, is there a way for me to create a cross region read replica with boto_rds.create_read_replica

**Describe the solution you'd like**
not sure if we can add source-region in salt module for this feature
**Describe alternatives you've considered**

**Additional context**

**Please Note**
If this feature request would be considered a substantial change or addition, this should go through a SEP process here https://github.com/saltstack/salt-enhancement-proposals, instead of a feature request.

**Comment by welcome** [comment]
Hi there! Welcome to the Salt Community! Thank you for making your first contribution. We have a lengthy process for issues and PRs. Someone from the Core Team will follow up as soon as possible. In the meantime, here’s some information that may help as you continue your Salt journey.
Please be sure to review our [Code of Conduct](https://github.com/saltstack/salt/blob/master/CODE_OF_CONDUCT.md). Also, check out some of our community resources including:

  - [Community Wiki](https://github.com/saltstack/community/wiki)
  - [Salt’s Contributor Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html)
  - [Join our Community Slack](https://join.slack.com/t/saltstackcommunity/shared_invite/zt-3av8jjyf-oBQ2M0vhXOhJpNpRkPWBvg)
  - [IRC on LiberaChat](https://web.libera.chat/#salt)
  - [Salt Project YouTube channel](https://www.youtube.com/channel/UCpveTIucFx9ljGelW63-BWg)
  - [Salt Project Twitch channel](https://www.twitch.tv/saltprojectoss)

There are lots of ways to get involved in our community. Every month, there are around a dozen opportunities to meet with other contributors and the Salt Core team and collaborate in real time. The best way to keep track is by subscribing to the Salt Community Events Calendar.
If you have additional questions, email us at saltproject@vmware.com. We’re glad you’ve joined our community and look forward to doing awesome things with you!
